torch/csrc/LazyTensor.cpp

```diff
@@ -0,0 +1,300 @@
+#include <ATen/core/LazyTensor.h>
+#include <torch/csrc/autograd/function.h>
+#include <torch/csrc/autograd/variable.h>
+#include <torch/csrc/jit/autodiff.h>
+#include <torch/csrc/jit/graph_executor.h>
+#include <torch/csrc/jit/interpreter.h>
+#include <torch/csrc/jit/operator.h>
+#include <torch/csrc/jit/script/function_schema_parser.h>
+
+namespace at {
+
+std::pair<torch::jit::Value*, std::vector<const LazyTensorImpl*>> getValue(
```

Comment:
This function is called `getValue`. The `torch::jit::Value*` output makes sense. So... what's the vector of lazy tensor impl pointers then?